src/main/java/com/delphix/masking/initializer/maskingApi/BackupDriver.java

```diff
@@ -448,6 +462,24 @@ private void backupProfiles() throws ApiCallException {
         maskingSetup.setProfileSets(getProfileSets.getProfileSets());
     }
 
+    /**
+     * Backup all profile expressions and sets
+     *
+     * @throws ApiCallException
+     */
+    private void backupMounts() throws ApiCallException {
+        GetMountInformations getMountInformations = new GetMountInformations();
+        try {
+            apiCallDriver.makeGetCall(getMountInformations);
+            if(getMountInformations.getMountInformations() != null) {
+                maskingSetup.setMountInformations(getMountInformations.getMountInformations());
+                getMountInformations.getMountInformations().forEach(mountInformation -> mountIdToName.put(mountInformation.getMountId(), mountInformation.getMountName()));
+            }
+        } catch (Exception e){
+
```

Comment:
The problem is in the case of MAC based masking engine, it will always fail. So I ignored the exception.

Reply:
Customers will not be using this on the mac, and if it does fail for a real reason, there will be no indication of this failure and they will be lead to believe that it succeeded.

If there are certain cases where there is a need to swallow the exceptions, we should add a flag so that users are explicitly giving permission to swallow it.

Reply:
Used the ignoreErrors application flag here.